alerting/src/main/resources/org/opensearch/alerting/alerts/finding_mapping.json

```diff
@@ -51,6 +51,15 @@
     },
     "timestamp": {
       "type": "long"
+    },
```

Comment:
We need to update the schema version when the schema is updated

Reply:
updated schema version